_includes/pbc/all/install-features/202311.0/install-the-data-exchange-api.md

```diff
@@ -216,7 +216,22 @@ class DynamicEntityGuiConfig extends SprykerDynamicEntityGuiConfig
             "validation": { "isRequired": false }
           }
         ]
-      }
+      },
+      "child_relations": [
+        {
+          "name": "countryTaxRates",
+          "is_editable": true,
+          "child_dynamic_entity_configuration": {
+            "table_name": "spy_tax_rate"
```

Comment:
so we have a direct link to the table and not to existing dynamic entity configuration?

Reply:
Yes, because we can't refer to ID, and the configuration table alias can be changed.
Do you think it makes sense?

Reply:
Requested to change this in https://github.com/spryker/suite-nonsplit/pull/8527#issuecomment-1866707517 to table alias, it should be more flexible for the case when we have several configurations per one table, thanks